src/components/WebexSettings/Tabs.jsx

```diff
@@ -18,22 +19,52 @@ import webexComponentClasses from '../helpers';
  * @param {string} props.selected  Specifies which tab is active
  * @param {object} props.style  Custom style to apply
  * @param {Tab[]} props.tabs  The tabs to be rendered
+ * @param {number} props.tabIndex  Tab index
```

Comment:
a little confusing, this is the html attribute, right? maybe "Tab index html attribute"